### PR TITLE
[Excel] (ExcelApi 1.14) ExcelApi 1.14 requirement set updates

### PR DESCRIPTION
**Pull request by alison-mk**
- Adding ExcelApi 1.14 requirement set article
- Updating Preview APIs article
- Updating Online-only APIs article
- Updating Availability and Requirement Sets articles

### CODE REVIEW

File: docs/reference/requirement-sets/excel-api-1-14-requirement-set.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+title: Excel JavaScript API requirement set 1.14
+description: 'Details about the ExcelApi 1.14 requirement set.'
+ms.date: 10/13/2021
+ms.prod: excel
+ms.localizationpriority: medium
+---
+
+# What's new in Excel JavaScript API 1.14
+
+The ExcelApi 1.14 added objects to control the data table feature of a chart, a method to locate all the precedent cells of a formula, and worksheet protection events to track changes to the protection state of a worksheet. It also added multiple `getItemOrNullObject` methods for objects like `CommentCollection`, `ShapeCollection`, and `StyleCollection` to improve error handling.
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/excel-api-1-14-requirement-set.md:11]: Consider linking to the section explaining the `orNullObject` pattern. (Also, please double-check that the link I provided is correct).
```suggestion
The ExcelApi 1.14 added objects to control the data table feature of a chart, a method to locate all the precedent cells of a formula, and worksheet protection events to track changes to the protection state of a worksheet. It also added multiple [`getItemOrNullObject`](../../develop/application-specific-api-model.md#ornullobject-methods-and-properties) methods for objects like `CommentCollection`, `ShapeCollection`, and `StyleCollection` to improve error handling.
```

File: docs/reference/requirement-sets/excel-api-1-14-requirement-set.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+| Feature area | Description | Relevant objects |
+|:--- |:--- |:--- |
+| Chart data tables | Control appearance, formatting, and visibility of data tables on charts. | [Chart](/javascript/api/excel/excel.chart), [ChartDataTable](/javascript/api/excel/excel.chartdatatable), [ChartDataTableFormat](/javascript/api/excel/excel.chartdatatableformat) |
+| Formula precedents | Return all the precedent cells of a formula. | [Range](/javascript/api/excel/excel.range) |
+| Queries | Retrieve query attributes like name, refresh date, and query count. | [Query](/javascript/api/excel/excel.query), [QueryCollection](/javascript/api/excel/excel.querycollection)|
```
Comment (AlexJerabek) [inline docs/reference/requirement-sets/excel-api-1-14-requirement-set.md:17]: Consider highlighting the Power Query integration.
```suggestion
| Queries | Retrieve Power Query attributes like name, refresh date, and query count. | [Query](/javascript/api/excel/excel.query), [QueryCollection](/javascript/api/excel/excel.querycollection)|
```

**Comment by alison-mk** [comment]
@AlexJerabek Thanks for the review! I'm blocked on the excel-api-requirement-sets page update until next week, but I've updated the other page you mentioned. Ready for re-review anytime.

File: docs/overview/office-add-in-availability.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ To work as expected, your Office Add-in might depend on a specific Office applic
...
...hared-runtime-requirement-sets.md)|- [BindingEvents](../reference/requirement-sets/office-add-in-requirement-sets.md#bindingevents)<br>- [CompressedFile](../reference/requirement-sets/office-add-in-requirement-sets.md#compressedfile)<br>- [DocumentEvents](../reference/requirement-sets/office-add-in-requirement-sets.md#documentevents)<br>- [File](../reference/requirement-sets/office-add-in-requirement-sets.md#file)<br>- [MatrixBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#matrixbindings)<br>- [MatrixCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#matrixcoercion)<br>- [Selection](../reference/requirement-sets/office-add-in-requirement-sets.md#selection)<br>- [Settings](../reference/requirement-sets/office-add-in-requirement-sets.md#settings)<br>- [TableBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#tablebindings)<br>- [TableCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#tablecoercion)<br>- [TextBindings](../reference/requirement-sets/office-add-in-requirement-sets.md#textbindings)<br>- [TextCoercion](../reference/requirement-sets/office-add-in-requirement-sets.md#textcoercion)|
```
Comment (AlexJerabek) [inline docs/overview/office-add-in-availability.md:24]: Did 1.14 make the 2021 cut off? I had assumed it hadn't when we updated the docs elsewhere. If 1.14 made the cut, please edit the line "Office 2021 supports ExcelApi 1.13 and earlier." on this page https://docs.microsoft.com/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets

Comment (alison-mk) [reply docs/overview/office-add-in-availability.md:24]: Confirmed that Office 2021 supports ExcelApi 1.14. I'll make the change on the page you linked.

**Comment by alison-mk** [comment]
@AlexJerabek Thanks for approving! I'll update the excel-api-requirement-sets article with the ExcelApi 1.14 build info once I have that info from the PM, hopefully in the next few days.

**Pull request merged**